scripts/openmc-make-depletion-chain-casl

```diff
@@ -0,0 +1,240 @@
+                        chain.reactions.append(name)
+
+                    if daughter not in decay_data:
+                        missing_rx_product.append((parent, name, daughter))
+                        daughter = 'Nothing'
+
+                    # Store Q value
+                    for mt in sorted(mts):
+                        if mt in reactions[parent]:
+                            q_value = reactions[parent][mt]
+                            break
+                    else:
+                        q_value = 0.0
+
+                    nuclide.reactions.append(ReactionTuple(
+                        name, daughter, q_value, 1.0))
+
+            if any(mt in reactions_available for mt in [18, 19, 20, 21, 38]):
+                if parent in fpy_data:
+                    q_value = reactions[parent][18]
+                    nuclide.reactions.append(
+                        ReactionTuple('fission', 0, q_value, 1.0))
+
+                    if 'fission' not in chain.reactions:
+                        chain.reactions.append('fission')
+                else:
+                    missing_fpy.append(parent)
+
+        if parent in fpy_data:
+            fpy = fpy_data[parent]
+
+            if fpy.energies is not None:
+                nuclide.yield_energies = fpy.energies
+            else:
+                nuclide.yield_energies = [0.0]
+
+            for E, table_yd, table_yc in zip(nuclide.yield_energies, fpy.independent, fpy.cumulative):
+                yields = defaultdict(float)
+                for product in table_yd:
+                    if product in decay_data:
+                        # identifier
+                        ifpy = CASL_CHAIN[product][2]
+                        # 1 for independent
+                        if ifpy == 1:
+                            if product not in table_yd:
+                                print('No independent fission yields found for {} in {}'.format(product, parent))
+                            else:
+                                yields[product] += table_yd[product].nominal_value
+                        # 2 for cumulative
+                        elif ifpy == 2:
+                            if product not in table_yc:
+                                print('No cumulative fission yields found for {} in {}'.format(product, parent))
+                            else:
+                                yields[product] += table_yc[product].nominal_value
+                        # -1 for stable + unstable
+                        elif ifpy == -1:
+                            if product not in table_yd:
+                                print('No independent fission yields found for {} in {}'.format(product, parent))
+                            else:
+                                yields[product] += table_yc[product].nominal_value
+                                product_meta = '{}_m1'.format(product)
+                                if product_meta in table_yd:
```

Comment:
Question: Looks like you're checking if `product_meta` is in `table_yd`, but you're actually indexing `product_meta` into `table_yc` below.  These two probably have identical keys in practice, but just wanted to check that this is the intended behavior.